src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,31 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.InvalidDataException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final byte MIN_LOG_PASSWORD_LENGTH = 6;
+    private static final byte MIN_AGE = 18;
```

Comment:
Consider using int instead of byte for these constants. While byte may save a bit of memory, int is the default type for integer literals in Java, and using int can avoid unnecessary type conversions.